pegasus/sites.v3/code.org/views/hoc2018_curriculum_banner.haml

```diff
@@ -0,0 +1,5 @@
+.bigbanner{style: "width: 100%; padding: 5px; background-color: #7665a0; text-align: center; color: white; padding: 20px; font-size: 15px"}
+  Go further with
```

Comment:
Do we need these strings to be translatable?

Reply:
Good catch.  Will fix that momentarily.